From 5f5b38d1480fbfe9a50ae63f72effc6dfeda05ac Mon Sep 17 00:00:00 2001
Message-Id: <5f5b38d1480fbfe9a50ae63f72effc6dfeda05ac.1478364305.git.kevin.kofler@chello.at>
From: Kevin Kofler <kevin.kofler@chello.at>
Date: Sat, 5 Nov 2016 17:41:38 +0100
Subject: [PATCH] [bootloader] Fix vfat_correct_case.

If the directory already existed, vfat_correct_case was returning only
the last part rather than the full path as it is supposed to.
---
 src/modules/bootloader/main.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/modules/bootloader/main.py b/src/modules/bootloader/main.py
index e367eeb..c009443 100644
--- a/src/modules/bootloader/main.py
+++ b/src/modules/bootloader/main.py
@@ -253,7 +253,7 @@ def install_grub(efi_directory, fw_type):
 def vfat_correct_case(parent, name):
     for candidate in os.listdir(parent):
         if name.lower() == candidate.lower():
-            return candidate
+            return os.path.join(parent, candidate)
     return os.path.join(parent, name)
 
 
-- 
2.7.4

